configure

```diff
@@ -201,8 +201,9 @@ while [ $# -ne 0 ]; do
     --with-doxygen=*)
       append_cache_entry Doxygen_ROOT_DIR PATH "$optarg"
       ;;
-    --with-jemalloc)
+    --with-jemalloc=*)
```

Comment:
Now I can't just `./configure --with-jemalloc`, and am required to always set the path, despite `Findjemalloc.cmake` finding it just fine without me specifying it.

Reply:
I hoped that `./configure --with=jemalloc=` would just work.

Reply:
```suggestion
    --with-jemalloc|--with-jemalloc=*)
```

This works as expected.

Note that this is not working as you'd like it to work:

```
./configure --with-jemalloc="$(brew --prefix jemalloc)"

[...]

-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    jemalloc_ROOT_DIR

[...]
```